bv-java-grupa-1/test/iohomework/ProcessCountryDataTest.java

```diff
@@ -0,0 +1,22 @@
+import customexception.CountryNotFoundException;
+import dataprocessing.ProcessCountryData;
+import org.junit.Test;
+
+public class ProcessCountryDataTest {
+
+    @Test(expected = CountryNotFoundException.class)
+    public void printCountryInfoTestException() throws CountryNotFoundException {
+        ProcessCountryData processCountryData = new ProcessCountryData();
+        processCountryData.printCountryInfo("country");
+    }
+
+    @Test
+    public void printCountryInfoTestNoException() {
```

Comment:
The unit test above `printCountryInfoTestException` is indeed necessary. However, the current unit test can't contain an assertion because we only use the `printCountryInfo` method for side effects (i.e. printing at the standard output the information about the given country). In this case, no automated tool can say anything about the result of this method.